docs/introduction/console-commands-for-application-management-phing-targets.md

```diff
@@ -106,8 +114,8 @@ Useful in a development environment and during deploying to production.
 
 #### backend-api-oauth-keys-generate
 
-Generates OAuth2 private and public key to directory `app/config/packages/oauth2/`, be careful never commit private keys.
-Generates also encryption key to file `app/config/packages/oauth2/parameters_oauth.yml`.
+Generates OAuth2 private and public key to directory `config/oauth2/`, be careful never commit private keys.
```

Comment:
#OT
Didn't know that. Just an idea, it would be nice to come up with a solution for it, so programmer doesn't need to worry about committing private key.
Maybe introduce a sub folder  `config/oauth2/private` for private keys, which will be registered in `.gitignore` or something like that.

Reply:
when backend-api is installed, in `config/oauth2` is .gitignore file with ignored public.key and private.key. Under these names are generated keys from phing target `backend-api--oauth-keys-generate`. So by default, it's covered and this last sentence is merely educational